MIG-1673: Release notes for MTC 1.7.18 (#86689)
openshift/openshift-docs: modules/migration-mtc-release-notes-1-7-18.adoc

// Module included in the following assemblies:
//
// * migration_toolkit_for_containers/mtc-release-notes-1-7.adoc

:_mod-docs-content-type: REFERENCE
[id="migration-mtc-release-notes-1-7-18_{context}"]
= {mtc-full} 1.7.18 release notes

> {mtc-first} 1.7.18 is a Container Grade Only (CGO) release, released to refresh the health grades of the containers, with no changes to any code in the product itself compared to that of {mtc-short} 1.7.17.

Reviewer: This is a run-on sentence. 

Rewrite as: "{mtc-first} 1.7.18 is a Container Grade Only (CGO) release, which is released to refresh the health grades of the containers, with no changes to any code in the product itself compared to that of {mtc-short} 1.7.17."

> {mtc-first} 1.7.18 is a Container Grade Only (CGO) release, released to refresh the health grades of the containers, with no changes to any code in the product itself compared to that of {mtc-short} 1.7.17.

Author: ```suggestion
{mtc-first} 1.7.18 is a Container Grade Only (CGO) release, which is released to refresh the health grades of the containers, with no changes to any code in the product itself compared to that of {mtc-short} 1.7.17.
```

> {mtc-first} 1.7.18 is a Container Grade Only (CGO) release, which is released to refresh the health grades of the containers, with no changes to any code in the product itself compared to that of {mtc-short} 1.7.17.

Reviewer 2: ```suggestion
{mtc-first} 1.7.18 is a Container Grade Only (CGO) release, which is released to refresh the health grades of the containers. No code was changed in the product itself compared to that of {mtc-short} 1.7.17.
```
even better, but not required for merge; we do need to know which branches, though, see my comment
